notebooks/01_prepare_data/wikidata.ipynb

```diff
@@ -0,0 +1,369 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set the environment path to find Recommenders\n",
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
+    "\n",
+    "import pandas as pd\n",
```

Comment:
Add 
```
print("System version: {}".format(sys.version))
print("Spark version: {}".format(pyspark.__version__))
```
for system environment checking. `Spark` checking is not needed in this notebook.